### PR TITLE
Change all grays on the page to abide by color contrast accessibility…

### PR DESCRIPTION
**Pull request by ibrand**
… standards

This fixes #896 

Here's what it looks like:
screen shot 2017-07-05 at 8 18 11 pm

Here's a GIF of the hover effect:
hovereffect

### CODE REVIEW

File: src/css/application.css [hunk 1]
```diff
@@ -420,13 +421,13 @@ body {
 }
 
 .editors__column-divider {
-  background-color: var(--color-dark-gray);
+  background-color: #aaa;
```
Comment (outoftime) [inline src/css/application.css]: can we give this a variable name too?

Comment (ibrand) [inline src/css/application.css]: I actually considered this but then wasn't sure what to name it..? like `--color-dark-gray-lighter`? heh. Any thoughts on that one?

Comment (ibrand) [inline src/css/application.css]: Wow totally forgot about this... any chance you have thoughts on this @outoftime ?

Comment (outoftime) [inline src/css/application.css]: @ibrand would it make sense to label the grays that _don’t_ provide sufficient contrast, to make clear that they should only be used for accent/aesthetics?

E.g.:

```css
  --color-low-contrast-gray: #eee;
  --color-light-gray: #aaa;
  --color-gray: #666;
  --color-dark-gray: #444;
```

**Comment by outoftime** [comment]
@ibrand this is so close and very good! think we could bring it home?

**Comment by ibrand** [comment]
Hey @outoftime, sorry for falling off the bandwagon here. I'll take a look this weekend and wrap it up.

_Sent from my Blackberry BBB100-1 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by ibrand** [comment]
Hey @outoftime , sorry for falling off the bandwagon here. I'll take a look this weekend and wrap it up.

_Sent from my Blackberry BBB100-1 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by ibrand** [comment]
Hey @outoftime , sorry for falling off the bandwagon here. I'll take a look this weekend and wrap it up.

**Comment by outoftime** [comment]
@ibrand awesome!

Something to think about—as I’ve been going through the redesign, I’ve generally found myself wondering whether I’m adding more accessibility problems with my choice of grays. So if we could find a way to make explicit what’s good / what’s bad (maybe via the names of the gray variables) that would be really cool!!

**Comment by ibrand** [comment]
Seems like there's been a lot of redesign type code written since I started this. Feeling a little shaky on the merge conflicts, but the code here feels pretty good. Any thoughts on that?

**Comment by outoftime** [comment]
@ibrand i hear you. i think the easiest thing would be for you to [give me push permissions to this branch](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/)? I’m down to rebase and deal with the conflicts : )

**Comment by ibrand** [comment]
screen shot 2017-08-27 at 9 26 11 pm

Should be good to go! Let me know if you need anything else, and thanks for offering to do this!

**Comment by outoftime** [comment]
@ibrand OK so I took a crack at merging in master, but I’m not super-confident that the outcome is entirely right, since it’s not purely a matter of reconciling before and after (some things should change). Would you mind pulling what’s here now and making sure I didn’t inadvertently set up any new accessibility issues?

**Comment by ibrand** [comment]
@outoftime Thanks for doing the merge! Took a quick pass and everything seems reasonable to me! The fonts are the part that actually matter and they're all dark. I'd say go for the merge.

**Pull request merged**